src/components/Menus/MainMenu/useThemeSwitchTabs.tsx

```diff
@@ -37,10 +41,49 @@ export const useThemeSwitchTabs = () => {
     setThemeMode(mode);
   };
 
+  // tooltips:
+  const lightModeTooltip =
```

Comment:
What is that doing exactly? seems to be extremely complicated and difficult to maintain

Reply:
Yes, there is quite some logic behind this. If you have any ideas on how to improve, I´d love to hear. We need to have three states of tooltips:
- "switch to theme" (if available)
- "theme is disabled" (if not available)
- no tooltip (if mode is already active)

--> Those three options of tooltips occur on the three items: light / dark / auto